lib/airbrake/action_cable.rb

```diff
@@ -0,0 +1,20 @@
+module ActionCable
+  module Channel
+    class Base
+      alias perform_action_without_airbrake perform_action
+
+      def perform_action(*args, &block)
+        perform_action_without_airbrake(*args, &block)
+      rescue Exception => exception # rubocop:disable Lint/RescueException
+        Airbrake.notify(exception) do |notice|
+          notice.stash[:rack_request] = ActionDispatch::Request.new(connection.env)
+          notice.stash[:action_cable_connection] = connection
```

Comment:
I think we should stash connection so You can add filter and take out intentifiers you wish to report with exception. And indentifiers on connection are in project's interests, not Airbrake's.